### PR TITLE
Add optional argument to specify source Queue

### PR DESCRIPTION
**Pull request by jaypea**
- add optional argument to specify source Queue
 - rename `activeQueue` to `destinationQueue` and `deadLetterQueue` to `sourceQueue` to make it more clear what is actually doing, it moves messages from one queue to another regardsless of their semantics.

### CODE REVIEW

File: sqs-dead-letter-requeue/main.go [hunk 1]
```diff
@@ -11,15 +11,20 @@ import (
...
 )
 
 func main() {
 	kingpin.MustParse(app.Parse(os.Args[1:]))
 
-	activeQueueName := *queueName
-
-	var deadLetterQueueName = activeQueueName + "_dead_letter"
+	destinationQueueName := *queueName
+
```
Comment (Partyschaum) [inline sqs-dead-letter-requeue/main.go]: You need to initialize the `sourceQueueName` variable first.
Insert a `var sourceQueueName string` here.

Comment (Partyschaum) [inline sqs-dead-letter-requeue/main.go]: ...nope, you don't due to this `:=` thing... OMG my go skills have died a slow and painfull death. 😢

**Comment by bracki** [comment]
Wouldn't it make sense to rename the CLI args as well? src and dst or similar?

**Comment by HenrikFricke** [comment]
I did a simple test to make sure everything works as expected. I created a `requeue-source` and a `requeue-dest` queue, added messages in the source queue and run the script. After some seconds all messages were moved to the `requeue-dest` queue. 🎉

**Pull request merged**